### ISSUE TITLE
Upon setup, credentials are requested not in-app, but in a webview, and never 'returned'

### ISSUE THREAD

**Issue by brenthuisman** [nextcloud/android#6394]
### Steps to reproduce
1. Enter a server
2. Rather than being allowed to enter my credentials in app (as per official docs), a webview opens on the web login of the server. See attached screenshot.
![signal-2020-06-29-161358](https://user-images.githubusercontent.com/2943652/86123365-a8bf4b80-bad9-11ea-8249-f85f5282efa5.png)
3. Logging in in the webview forwards me to the website of the server in the webview, without any way to return to the app. Closing the view closes the app, and no settings are stored from this interactions.

### Expected behaviour
- Allowing me to enter credentials in-app and setup shared folders etc.

### Actual behaviour
- Webview to website of server is opened.

### Environment data
Android version: 10

Device model: Pixel

Stock or customized system: LineageOS + Nextcloud from F-Droid

Nextcloud app version: latest (as of 28-06-2020)

Nextcloud server version: not certain

Reverse proxy: not certain

### Logs
none available

**Comment by tobiasKaminsky** [comment]
After entering the credentials it should return and login.
You can test it with try.nextcloud.com to see how it should work.
If this then works, but not on your server, I supect a sever configuration problem.

**Comment by smalahove** [comment]
I have this problem, too. Trying to set up two new phones (both Android 10), I am only presented with the web view after logging in.

Using Nextcloud Android client v3.12.0 and ownCloud server 10.3.2.2, I presumed it might be something with the connection to a non-Nextcloud server?

We also have two older phones where the Android client works as expected, one of them is also a Android 10. I guess these continue to work since they have already authenticated and can keep working in webdav-mode or something?

**Comment by tobiasKaminsky** [comment]
@smalahove this is about Nextcloud…ownCloud was partially "supported" until a while ago, but as they do not have this weblogin auth it will not work.

@brenthuisman do you use a Nextcloud server?

**Comment by brenthuisman** [comment]
Nope, the server is not Nextcloud. I guess that's the reason then, no webauth. Is that the only auth Nextcloud Android supports? Because the docs don't show/state this.

**Comment by smalahove** [comment]
> @smalahove this is about Nextcloud…ownCloud was partially "supported" until a while ago, but as they do not have this weblogin auth it will not work.

Thanks, I failed to find this information anywhere, and it makes perfect sense.

I tested now with an older version of the Android client from the time we setup the aforementioned Android 10 phone, and found v3.7.0 warned of end-of-life for NC13. With this I could authenticate to ownCloud. A subsequent upgrade of the client also works (no need to login anew), explaining why this was true for the 2 phones I mentioned above.

Anyway, this was a good reminder for me to migrate to NC sooner than later.
